lib/proto/handle-response.js

```diff
@@ -0,0 +1,72 @@
+var clone = require("../util/clone.js")
+var newUid = require("../uniqueid.js")
+var trigger = require("../events/trigger.js")
+
+module.exports = function(responseText, request, href) {
+  // Fail if unable to load HTML via AJAX
+  if (responseText === false) {
+    var tempOptions = this.options
```

Comment:
This needs to be `tempOptions = clone(this.options)`, or you'll mutate the current instance's options

Reply:
Good point.